go.mod

```diff
@@ -21,10 +21,10 @@ require (
 	github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0
 	github.com/keybase/go-ps v0.0.0-20190827175125-91aafc93ba19
 	github.com/michaelquigley/pfxlog v0.0.0-20190813191113-2be43bd0dccc
-	github.com/netfoundry/ziti-edge v0.10.3
-	github.com/netfoundry/ziti-fabric v0.9.5
+	github.com/netfoundry/ziti-edge v0.11.0
+	github.com/netfoundry/ziti-fabric v0.10.0
```

Comment:
Fabric at 10.0?

Reply:
fabric versions will be lagging behind because we don't push bumps upstream only down